packit_service/worker/handlers/github_handlers.py

```diff
@@ -154,13 +163,26 @@ def run(self) -> TaskResults:
         for branch in get_branches(
             *self.job_config.metadata.dist_git_branches, default="master"
         ):
-            try:
-                self.api.sync_release(
-                    dist_git_branch=branch, version=self.data.tag_name
-                )
-            except Exception as ex:
-                sentry_integration.send_to_sentry(ex)
-                errors[branch] = str(ex)
+            exception: Optional[Exception] = None
+            for _ in range(RETRY_LIMIT):
```

Comment:
if I understand this code correctly, it's trying to fetch archive for 50 seconds

I'd say it's better design to utilize celery here and instead of doing sleep(), use celery's retry